server/src/com/cloud/server/ManagementServerImpl.java

```diff
@@ -3567,7 +3567,17 @@ public boolean deleteSSHKeyPair(final DeleteSSHKeyPairCmd cmd) {
         final Long domainId = cmd.getDomainId();
         final Long projectId = cmd.getProjectId();
 
-        final Account owner = _accountMgr.finalizeOwner(caller, accountName, domainId, projectId);
+        Account owner = null;
```

Comment:
Hello @ustcweizhou 

I think this piece of code could be extracted from the deleteSSHKeyPair method. It has gotten a little big from it's past form and further improvements could be done outside of the main "deleteSSHKeyPair" flow if needed.